torch/nn/modules/module.py

```diff
@@ -165,6 +158,10 @@ def register_module_full_backward_hook(
     in :attr:`grad_input` and :attr:`grad_output` will be ``None`` for all non-Tensor
     arguments.
 
+    Note that this hook need to apply a Function to the inputs and outputs of the Module
+    during the forward pass. This means that Tensors passed as argument to the Module and
+    the ones received by the forward function will not be the same, but views of each other.
```

Comment:
More precisely, I think this paragraph can be summarized as, 

"For technical reasons, when this hook is applied to a module its forward function will receive a view of each tensor passed to the module."

Reply:
Great suggestion. Updated.